services/wallet/transfer/helpers.go

```diff
@@ -0,0 +1,245 @@
+package transfer
+
+import (
+	"database/sql"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	ethTypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/status-im/status-go/account"
+	"github.com/status-im/status-go/eth-node/crypto"
+	"github.com/status-im/status-go/eth-node/types"
+	"github.com/status-im/status-go/services/wallet/bridge"
+	wallet_common "github.com/status-im/status-go/services/wallet/common"
+)
+
+func rowsToMultiTransactions(rows *sql.Rows) ([]*MultiTransaction, error) {
+	var multiTransactions []*MultiTransaction
+	for rows.Next() {
+		multiTransaction := &MultiTransaction{}
+		var fromAmountDB, toAmountDB sql.NullString
+		var fromTxHash, toTxHash sql.RawBytes
+		err := rows.Scan(
+			&multiTransaction.ID,
+			&multiTransaction.FromNetworkID,
+			&fromTxHash,
+			&multiTransaction.FromAddress,
+			&multiTransaction.FromAsset,
+			&fromAmountDB,
+			&multiTransaction.ToNetworkID,
+			&toTxHash,
+			&multiTransaction.ToAddress,
+			&multiTransaction.ToAsset,
+			&toAmountDB,
+			&multiTransaction.Type,
+			&multiTransaction.CrossTxID,
+			&multiTransaction.Timestamp,
+		)
+		if len(fromTxHash) > 0 {
+			multiTransaction.FromTxHash = common.BytesToHash(fromTxHash)
+		}
+		if len(toTxHash) > 0 {
+			multiTransaction.ToTxHash = common.BytesToHash(toTxHash)
+		}
+		if err != nil {
+			return nil, err
+		}
+
+		if fromAmountDB.Valid {
+			multiTransaction.FromAmount = new(hexutil.Big)
+			if _, ok := (*big.Int)(multiTransaction.FromAmount).SetString(fromAmountDB.String, 0); !ok {
+				return nil, errors.New("failed to convert fromAmountDB.String to big.Int: " + fromAmountDB.String)
+			}
+		}
+
+		if toAmountDB.Valid {
+			multiTransaction.ToAmount = new(hexutil.Big)
+			if _, ok := (*big.Int)(multiTransaction.ToAmount).SetString(toAmountDB.String, 0); !ok {
+				return nil, errors.New("failed to convert fromAmountDB.String to big.Int: " + toAmountDB.String)
+			}
+		}
+
+		multiTransactions = append(multiTransactions, multiTransaction)
+	}
+
+	return multiTransactions, nil
+}
+
+func (tm *TransactionManager) ensureTransactionsSigned(signatures map[string]SignatureDetails) error {
```

Comment:
@IvanBelyakoff could you please rename this function to `addSignaturesToTransacations` or `associateSignaturesToTransactions`, cause this function doesn't sign transactions?